beancount/utils/file_utils_test.py

```diff
@@ -42,23 +43,31 @@ def walk(fords):
         self.assertEqual(
             sorted(
                 [
-                    "alice/bottle.txt",
-                    "rabbit/suit/glasses.txt",
-                    "caterpillar/who-are-you.txt",
+                    Path(self.tempdir, "alice/bottle.txt"),
```

Comment:
I don't understand why this change is necessary.

Reply:
`walk` may yield abs path or rel path on different platform. And also, `os.sep` problem.

Reply:
```text
beancount/utils/file_utils_test.py::TestFileUtilsFind::test_find_files FAILED

========================================================================== FAILURES ===========================================================================
______________________________________________________________ TestFileUtilsFind.test_find_files ______________________________________________________________

self = <beancount.utils.file_utils_test.TestFileUtilsFind testMethod=test_find_files>

    def test_find_files(self):
        def walk(fords):
            return sorted(clean(self.tempdir, file_utils.find_files(fords)))

>       self.assertEqual(
            sorted(
                [
                    "alice/bottle.txt",
                    "rabbit/suit/glasses.txt",
                    "caterpillar/who-are-you.txt",
                ]
            ),
            walk([self.tempdir]),
        )
E       AssertionError: Lists differ: ['alice/bottle.txt', 'caterpillar/who-are-y[30 chars]txt'] != ['C:\\Users\\Trim21\\AppData\\Local\\Temp\\[241 chars]txt']
E
E       First differing element 0:
E       'alice/bottle.txt'
E       'C:\\Users\\Trim21\\AppData\\Local\\Temp\\[41 chars].txt'
E
E       - ['alice/bottle.txt', 'caterpillar/who-are-you.txt', 'rabbit/suit/glasses.txt']
E       + ['C:\\Users\\Trim21\\AppData\\Local\\Temp\\TestFileUtilsFind.d1t917_c\\alice\\bottle.txt',
E       +  'C:\\Users\\Trim21\\AppData\\Local\\Temp\\TestFileUtilsFind.d1t917_c\\caterpillar\\who-are-you.txt',
E       +  'C:\\Users\\Trim21\\AppData\\Local\\Temp\\TestFileUtilsFind.d1t917_c\\rabbit\\suit\\glasses.txt']

beancount\utils\file_utils_test.py:43: AssertionError
```

Reply:
Ah, looks like I should fix this in `clean` instead of tests